roles/internal/crayfish/tasks/install.yml

```diff
@@ -0,0 +1,92 @@
+---
+
+- name: Install requisite packages
+  apt:
+    name: "{{item}}"
+    state: present
+  with_items:
+    - tesseract-ocr
```

Comment:
You'll need to add imagemagick to this list.  @jonathangreen Now that we're using grok, is there any need to use Lyrasis's PPA?

Reply:
@jonathangreen bump

Reply:
Sorry. missed this one in the flood of islandora release tickets yesterday. 

Houdini is just a wrapper on imagemagick. We added grok to Cantaloupe. If we want Houdini to be able to work with grok or kdu, we'll have to do some work on it. That said there is certainly an argument that doing that could be beneficial, or that we should just totally dump Houdini and just use cantaloupe instead... but I think thats another ticket.

That was a long winded way of saying we need imagemagick, and if we want it compiled with the JP2 extensions, installing it from the Lyrasis PPA is probably the thing to do.

Reply:
@dannylamb I guess the downside would be that cantaloupe won't fit into the API-X model, but it is a image transformation service that we don't have to maintain...

Reply:
Just to be clear, the landing here is that I need to get Imagemagick from Lyrasis, correct?

Reply:
👍

Reply:
https://launchpad.net/~lyrasis/+archive/ubuntu/imagemagick-jp2